jac-uk/qt
# Bugfix/Fix NPM Vulnerabilities

## What's included?

[Identify And Fix NPM Vulnerabilities In All Repositories #2513](https://github.com/jac-uk/admin/issues/2513)

Fix the vulnerabilities in the QT and QT Admin.

**Note:** We might create a separate ticket to upgrade the Node version in the QT Digital Platform. 

## Who should test?
✅ Developers

## How to test?

1. Go to `qt` and `qt-admin` folder.
2. Run `npm audit` in the terminal and check if there are no high vulnerabilities.
